ui/raidboss/data/05-shb/raid/e7n.js

```diff
@@ -7,7 +7,253 @@
+      condition: function(data) {
+        return data.role == 'healer' || data.role == 'tank' || data.CanAddle();
+      },
+      response: Responses.aoe(),
+    },
+    {
+      id: 'E7N Unshadowed Stake',
+      regex: Regexes.tether({ source: 'The Idol Of Darkness', id: '0025' }),
+      regexFr: Regexes.tether({ source: 'Idole Des Ténèbres', id: '0025' }),
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'E7N Left With Thee',
+      regex: Regexes.gainsEffect({ effect: 'Left With Thee' }),
+      regexFr: Regexes.gainsEffect({ effect: 'Translation Gauche' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Teleporting Left',
+        fr: 'Téléportation à gauche',
+        ko: '왼쪽으로 순간이동',
+      },
+    },
+    {
+      id: 'E7N Right With Thee',
+      regex: Regexes.gainsEffect({ effect: 'Right With Thee' }),
+      regexFr: Regexes.gainsEffect({ effect: 'Translation Droite' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Teleporting Right',
+        fr: 'Téléportation à droite',
+        ko: '오른쪽으로 순간이동',
+      },
+    },
+    {
+      id: 'E7N Forward With Thee',
+      regex: Regexes.gainsEffect({ effect: 'Forward With Thee' }),
+      regexDe: Regexes.gainsEffect({ effect: 'Deportation: Vorne' }),
+      regexFr: Regexes.gainsEffect({ effect: 'Translation Avant' }),
+      regexJa: Regexes.gainsEffect({ effect: '強制転移：前' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Teleporting Forward',
+        de: 'Teleportation Vorwärts',
+        fr: 'Téléportation devant',
+        ko: '앞으로 순간이동',
+        cn: '向前传送',
+      },
+    },
+    {
+      id: 'E7N Back With Thee',
+      regex: Regexes.gainsEffect({ effect: 'Back With Thee' }),
+      regexDe: Regexes.gainsEffect({ effect: 'Deportation: Hinten' }),
+      regexFr: Regexes.gainsEffect({ effect: 'Translation Arrière' }),
+      regexJa: Regexes.gainsEffect({ effect: '強制転移：後' }),
+      condition: Conditions.targetIsYou(),
+      infoText: {
+        en: 'Teleporting Back',
+        de: 'Teleportation Rückwärts',
+        fr: 'Téléportation derrière',
+        ko: '뒤로 순간이동',
+        cn: '向后传送',
+      },
+    },
+    {
+      id: 'E7N Strength In Numbers Donut',
+      regex: Regexes.startsUsing({ source: 'Idolatry', id: '4C4C', capture: false }),
+      suppressSeconds: 1,
+      infoText: {
+        en: 'Teleport into donut',
+      },
+    },
+    {
+      // Ordinarily we might not warn on ground AoE markers. However, there are player-dropped
+      // markers just before this, so it might be difficult to see.
+      id: 'E7N Strength In Numbers Circle',
+      regex: Regexes.startsUsing({ source: 'Idolatry', id: '4C4D', capture: false }),
+      suppressSeconds: 1,
+      response: Responses.getOut(),
```

Comment:
Sounds good! I think "get out" is also a reasonable direction to give vs some sort of ambiguous "avoid those aoes on the ground" kind of thing.